plugins/cc/g_congruence.mlg

```diff
@@ -26,6 +26,11 @@ TACTIC EXTEND cc
 | [ "congruence" "with" ne_constr_list(l) ] -> { congruence_tac 1000 l }
 | [ "congruence" natural(n) "with" ne_constr_list(l) ] ->
    { congruence_tac n l }
+| [ "simple" "congruence" ] -> { simple_congruence_tac 1000 [] }
+| [ "simple" "congruence" natural(n) ] -> { simple_congruence_tac n [] }
+| [ "simple" "congruence" "with" ne_constr_list(l) ] -> { simple_congruence_tac 1000 l }
+| [ "simple" "congruence" natural(n) "with" ne_constr_list(l) ] ->
+   { simple_congruence_tac n l }
```

Comment:
@jfehrle Making those changes gives me the error:
```
$ make -f Makefile.make doc_gram_rsts
make --warn-undefined-variable --no-builtin-rules -f Makefile.build doc_gram_rsts
make[1]: Entering directory '/home/user/stdlib/simple_congruence'
MKDIR     BUILD_OUT
DUNE      sources
COQDEP    VFILES
DUNE      sources
DUNE      _build/default/doc/tools/docgram/doc_grammar.exe
DOC_GRAM
DOC_GRAM_RSTS
Error: Can't find 'global univ_annot' in edit for 'atomic_constr'
Error: Can't find 'global univ_annot' to DELETE for 'atomic_constr'
```

Reply:
Should I also change congruence akin to `| [ "simple" "congruence" natural_opt(n) ] -> { simple_congruence_tac (Option.default 1000 n) [] }` (and edit corresponding `common.edit_mlg` lines?

Reply:
>  (pps it requires `./configure`, besides the error, which I was told to avoid for dune builds)

Yeah, unfortunately, this build target is still limited to the legacy build system.

Reply:
> (ps is `("with" LIST1 constr )` correct, or rather `( "with" LIST1 constr )`?

They will work the same, but the second is easier to read.